models/articles.models.js

```diff
@@ -31,8 +31,20 @@ exports.selectArticleCommentsById = (article_id) => {
         ORDER BY created_at DESC;`,
         [article_id])
     .then(({rows}) => {
-        if (rows.length === 0) {
+        if (!rows.length) {
             return Promise.reject({status: 404, msg: 'Article not found and comments are unavailable'})
         } else return rows
     })
+}
+
+exports.addComment = (commentToPost, article_id) => {
+    return db.query(`INSERT INTO comments (author, body, article_id) VALUES ($1, $2, $3)
+        RETURNING *;`, [commentToPost.username, commentToPost.body, article_id])
+    .then(( { rows }) => {
+        if (!rows.length) {
+            return Promise.reject({status: 404, msg: 'Article not found and comment cannot be posted'})
+        } else if (rows[0].body === '') {
```

Comment:
lets handle this before we query the db because if the string is "" this is NOT NULL so it will be entered into the db 😄